OpenChange/MAPIStoreMailMessage.m

```diff
@@ -572,28 +572,21 @@ - (int) getPidTagMessageClass: (void **) data
   return MAPISTORE_SUCCESS;
 }
 
-/* Note: this applies to regular mails... */
-// - (int) getPidTagReplyRequested: (void **) data // TODO
-//                    inMemCtx: (TALLOC_CTX *) memCtx
-// {
-//   if (!headerSetup)
-//     [self _fetchHeaderData];
-
-//   return (mailIsEvent
-//           ? [self getYes: data inMemCtx: memCtx]
-//           : [self getNo: data inMemCtx: memCtx]);
-// }
-
-/* ... while this applies to invitations. */
-- (int) getPidTagResponseRequested: (void **) data // TODO
-                          inMemCtx: (TALLOC_CTX *) memCtx
+- (int) getPidTagReplyRequested: (void **) data
+		       inMemCtx: (TALLOC_CTX *) memCtx
 {
   if (!headerSetup)
     [self _fetchHeaderData];
 
   return (mailIsEvent
-          ? [self getNo: data inMemCtx: memCtx]
-          : MAPISTORE_ERR_NOT_FOUND);
+          ? [self getYes: data inMemCtx: memCtx]
+          : [self getNo: data inMemCtx: memCtx]);
```

Comment:
This second change is not required. If the property is absent, the response is same one than no